io.catenax.generic.digital_product_passport/7.0.0/DigitalProductPassport.ttl

```diff
@@ -0,0 +1,1805 @@
+
+:OperationEntity a samm:Entity ;
+   samm:preferredName "Operation Entity"@en ;
+   samm:description "Operational information of the product such as the owner, manufacturer and importer."@en ;
+   samm:properties ( :manufacturing :import [ samm:property :otherOperators; samm:optional true ] :extendedProducerResponsibilityScheme ) .
+
+:HandlingEntity a samm:Entity ;
+   samm:preferredName "Handling Entity"@en ;
+   samm:description "Entity to describe different aspects in relation with the handling of the product with attributes if applicable to the product."@en ;
+   samm:properties ( [ samm:property :spareParts; samm:optional true; samm:payloadName "content" ] :applicable ) .
+
+:ProductCharacteristicsEntity a samm:Entity ;
+   samm:preferredName "Product Characteristics Entity"@en ;
+   samm:description "Entity which defines specific characteristics of a product such as different life spans and physical dimensions."@en ;
+   samm:properties ( :lifespan :physicalDimension [ samm:property :physicalState; samm:optional true ] [ samm:property :generalPerformanceClass; samm:optional true ] [ samm:property :otherCharacteristics; samm:optional true ] ) .
+
+:CommercialEntity a samm:Entity ;
+   samm:preferredName "Commercial Entity"@en ;
+   samm:description "Commercial information."@en ;
+   samm:properties ( [ samm:property :placedOnMarket; samm:optional true ] :purpose [ samm:property :purchaseOrder; samm:optional true ] :recallInformation ) .
+
+:SustainabilityEntity a samm:Entity ;
+   samm:preferredName "Sustainability Entity"@en ;
+   samm:description "Entity with the properties regarding the sustainability of the product."@en ;
+   samm:properties ( :productStatus :productFootprint [ samm:property :reparabilityScore; samm:optional true ] [ samm:property :durabilityScore; samm:optional true ] :reusability :recycleabilityPerformance ) .
+
+:DocumentEntity a samm:Entity ;
+   samm:preferredName "Document Entity"@en ;
+   samm:description "Document entity with header, content, category and type."@en ;
+   samm:properties ( :content :category :contentType :header ) .
+
+:AdditionalDataEntity a samm:Entity ;
+   samm:preferredName "Additional Data Entity"@en ;
+   samm:description "Entity with additional data. Properties are label, description, type, data and children. Either the fields data or children filled. Children constructs a hierarchy with the same attributes. Data is filled as the end of an path/leaf."@en ;
+   samm:properties ( :label :description :type [ samm:property :data; samm:optional true ] [ samm:property :children; samm:optional true ] ) .
+
+:ComplianceEntity a samm:Entity ;
+   samm:preferredName "Compliance Entity"@en ;

```

Comment:
Do not use metamodel element names in description: just "compliance information" is sufficient

same for other elements